docs/getting-started/install/dreamhost.md

```diff
@@ -0,0 +1,160 @@
+# Install Dokku on DreamHost Cloud Server with cloud-init
+
+Cloud-init script can be used to automate installation of Dokku on
+Dreamhost (or any other OpenStack-compatible cloud with minimal
+changes).
+
+To create a new server and install Dokku from the openstack command
+line interface:
+
+    $ openstack server create \
```

Comment:
Also, remove the `$`. We properly show this as a shell command on our docs page via a helpful tag on the top-right of codeblocks as shown [here](http://dokku.viewdocs.io/dokku/getting-started/installation/).